features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/site/themes/responsive/templates/item-add/js/item-add.js

```diff
@@ -105,11 +105,12 @@ $( document ).ready(function() {
        });
     });
 
+    $('.toggleContainers .controls').hide();
     $('.toggleRadios input[type=radio]').prop('checked', false);
     $('.toggleRadios input[type=radio]').click(function(){
         $('.toggleContainers .controls').hide();
         $('.toggleRadios input[type=radio]').prop('checked', false);
-        $('#' + $(this).val()).closest('div').fadeIn();
+        $('#' + $(this).val()).parent().parent().fadeIn();
```

Comment:
Don't traverse elements using `parent()` api like this. Use `closest()` api instead